packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts

```diff
@@ -86,3 +86,355 @@ describe('Token', () => {
   })
 })
+
+describe('Consortium', () => {
+  const tGroup = TestingGroup.create(4)
+  let account0: string, account1: string, account2: string, account3: string
+  let idBTC: string, idDOGE: string
+  const symbolBTC = 'BTC'
+  const symbolDOGE = 'DOGE'
+  const blocksPerDay = (60 * 60 * 24) / (10 * 60) // 144 in regtest
+
+  beforeAll(async () => {
+    const startFlags: StartFlags[] = [{ name: 'regtest-minttoken-simulate-mainnet', value: 1 }]
+    await tGroup.start({ startFlags: startFlags })
+
+    account0 = await tGroup.get(0).generateAddress()
+    account1 = await tGroup.get(1).generateAddress()
+    account2 = await tGroup.get(2).generateAddress()
+    account3 = await tGroup.get(3).generateAddress()
+
+    await tGroup.get(0).token.create({
+      symbol: symbolBTC,
+      name: symbolBTC,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: account0
+    })
+    await tGroup.get(0).generate(1)
+
+    await tGroup.get(0).token.create({
+      symbol: symbolDOGE,
+      name: symbolDOGE,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: account0
+    })
+    await tGroup.get(0).generate(1)
+
+    await tGroup.get(0).container.fundAddress(account0, 10)
+    await tGroup.get(0).container.fundAddress(account1, 10)
+    await tGroup.get(0).container.fundAddress(account2, 10)
+    await tGroup.get(0).container.fundAddress(account3, 10)
+
+    await tGroup.get(0).generate(1)
+
+    idBTC = await tGroup.get(0).token.getTokenId(symbolBTC)
+    idDOGE = await tGroup.get(0).token.getTokenId(symbolDOGE)
+  })
+
+  afterAll(async () => {
+    await tGroup.stop()
+  })
+
+  async function setGovAttr (ATTRIBUTES: object): Promise<void> {
+    const hash = await tGroup.get(0).rpc.masternode.setGov({ ATTRIBUTES })
+    expect(hash).toBeTruthy()
+    await tGroup.get(0).generate(1)
+  }
+
+  it('should throw an error if foundation or consortium member authorization is not present', async () => {
+    const errorMsg = 'RpcApiError: \'Test MintTokenTx execution failed:\nYou are not a foundation member or token owner and cannot mint this token!\', code: -32600, method: minttokens'
+    await expect(tGroup.get(2).rpc.token.mintTokens(`1@${symbolBTC}`)).rejects.toThrow(errorMsg)
+    await expect(tGroup.get(3).rpc.token.mintTokens(`1@${symbolBTC}`)).rejects.toThrow(errorMsg)
+    await expect(tGroup.get(2).rpc.token.mintTokens(`1@${symbolDOGE}`)).rejects.toThrow(errorMsg)
+    await expect(tGroup.get(3).rpc.token.mintTokens(`1@${symbolDOGE}`)).rejects.toThrow(errorMsg)
+  })
+
+  it('should throw an error if the token is not specified in governance vars', async () => {
+    // Enable consortium
+    await setGovAttr({ 'v0/params/feature/consortium': 'true' })
+
+    // Set global consortium mint limit for BTC
+    await setGovAttr({
+      [`v0/consortium/${idBTC}/mint_limit`]: '10',
+      [`v0/consortium/${idBTC}/mint_limit_daily`]: '5'
+    })
```

Comment:
Your tests are written sequentially; you should avoid that. Each `it('')` should be side-effect free.

This is important to allow tests to be run independently, e.g., 'should throw an error if member daily mint limit exceeds' can't be run standalone; it requires all tests above to be run in sequential order.

```shell
jest --testNamePattern=^Consortium should throw an error if member daily mint limit exceeds --runTestsByPath /@JellyfishSDK/jellyfish/packages/jellyfish-api-core/__tests__/category/token/mintTokens.test.ts
```